Footer for egi whitelabel

jarekzet opened the pull request:
new footer layout for EGI whitelabel
https://github.com/cyfronet-fid/marketplace-egi/issues/3
https://marketplace.egi.eu/

Marcelinna commented:
we should delete element below footer ?
![image](https://user-images.githubusercontent.com/59872096/151771107-5a86c6aa-4d55-4f35-ab58-4c586c58dfde.png)

Marcelinna commented:
ticket
![image](https://user-images.githubusercontent.com/59872096/151774265-2d5feaca-87eb-418d-adba-92392a08aa90.png)

PR 
![image](https://user-images.githubusercontent.com/59872096/151774306-c619e6aa-bc60-4330-a637-624618bc53a0.png)

In PR is additionally added youtube, twiter logo - which redirect to EOSC media
In PR is added a link to the EGI logo which redirect to the EOSC page (https://eosc-portal.eu/)

should these elements be introduced?

lukasz-korzeniowski commented:
Twiter should be linked to: https://twitter.com/egi_einfra 
Please replace Youtube  with LinkedIn icon and link to: https://www.linkedin.com/company/egi-foundation

jarekzet commented:
Added mods on social media links:
Twiter: https://twitter.com/egi_einfra
LinkedIn: https://www.linkedin.com/company/egi-foundation

Removed link on logo in footer.

Marcelinna commented:
So we need link to: 
Contact us
Terms and conditions
SLA
Sitemap 
 My orders
My credit slips
My addresses
My personal info 
Privacy policy

Marcelinna commented on app/views/layouts/_navbar.html.haml:
```diff
@@ -7,17 +7,17 @@
             %a.simple-item{ href: "/profile" }= _("Profile")
           %li
-            = link_to _("Logout"), destroy_user_session_path, method: :delete, "data-e2e" => "logout",
-            class: "btn btn-primary font-weight-bold mt-5 logout-link"
+            = link_to _("Logout"), destroy_user_session_path, method: :delete,
+            class: "btn btn-primary font-weight-bold mt-5"
         - else
```
why you delete logout-link?

Marcelinna commented on app/views/layouts/_sections.html.haml:
```diff
@@ -21,7 +21,7 @@
-          %strong= link_to _("Logout"), destroy_user_session_path, method: :delete
+          %strong= link_to _("Logout"), destroy_user_session_path, method: :delete, "data-e2e" => "logout"
       - else
         %li
-          %strong= link_to _("Login"), user_checkin_omniauth_authorize_path
```
Please delete "data-e2e" => "my-eosc-button"
